sdcm/sct_runner.py

```diff
@@ -269,6 +276,7 @@ def create_instance(self, test_id: str, test_duration: int, region_az: str):
                 {"Key": "keep_action", "Value": "terminate"},
             ],
             region_az=region_az,
+            test_duration=test_duration,
```

Comment:
why did you add the tag? `keep` stores this information

Reply:
I didn't change the tags_list parameter. The test_duration is used inside _create_instance() -> self.instance_root_disk_size().